components/omega/doc/userGuide/OmegaBuild.md

```diff
@@ -0,0 +1,124 @@
+(omega-user-build)=
+
+Omega's build system is built upon the CMake build tool, which provides
+a robust foundation for managing the build process.
+
+The Omega build system supports two modes: standalone and E3SM component.
+
+## Standalone Build
+
+To perform a standalone build, you need to execute the cmake command with
```

Comment:
Can we add a sentence or two with the need to create a build directory. Maybe something like:

CMake and OMEGA prefer an out-of-source build.  This enables a user to build and maintain multiple executables from the same source directory. The standard practice is for a user to create a separate directory where the build should take place and the commands below should be launched from that directory.

Reply:
Sure, I will add the info in next commit.